en-US/snippets/2017/nov2017_mobile.lang

```diff
@@ -0,0 +1,243 @@
+## NOTE: Reference bug 1410997 and screenshots. Deadline: 10 November, 2017.
+## NOTE: Mobile contextual hints: character limitation includes spaces.
+## NOTE: Brand names stay in English except when noted.
+
+
+# Mobile Banners
+# Text limit: 79 characters, including spaces.
```

Comment:
You need to add the `## MAX_LENGTH` to all these strings, otherwise good luck in figuring out who's below limits.